src/NoFrixion.MoneyMoov/Enums/PayrunStatus.cs

```diff
@@ -20,10 +20,11 @@ public enum PayrunStatus
 {
     None = 0,
     Draft = 1,
-    Submitted = 2,
-    Completed = 3, 
-    Rejected = 4,
-    AuthorisationPending = 5,
-    PayoutsCreated = 6,
-    Edited = 7,
+    Queued = 2,
```

Comment:
It's better practice to append a new item to an enum.

Do we need a new Queued state? Plus what does queued mean exactly?

Reply:
@donalnofrixion I have appended the new status. Also, added proper summary to all the states. Thanks.